src/base/castlerandom.pas

```diff
@@ -47,11 +47,12 @@   TCastleRandom = class(TObject)
     function RandomBoolean: boolean;
     { Randomly provides "-1", "0" or "1" with equal chances. }
     function RandomSign: longint;
-    { Returns a random number in 0 .. High(LongWord) range. }
+    { Returns a random number in 1 .. High(LongWord) range. }
```

Comment:
Oooops. I've forgotten mentioning about randomInt64... and there's a problem we have here. its not 1...high(QWORD) but high(DWORD)...high(QWORD)! because we actually compose it by seed shl 32 + seed, and seed may never be zero in xorshift. 
I've thought about this problem before, but forgotten about it later. I'll see what I can do here.